Tests/SwiftBoxMetricsTests/MetricTypesTests.swift

```diff
@@ -20,13 +20,13 @@ class MetricTypesTests: XCTestCase {
 
     func testGaugeStatsDFormat() throws {
         var metric = GaugeMetric(name: "test", value: 2, type: .set)
-        XCTAssertEqual(metric.getStatsDLine(), "test:2|g")
+        XCTAssertEqual(metric.getStatsDLine(), "test:2.0|g")
```

Comment:
Split this unit test into 3, because we're testing 3 different logical / business assertions.